oxidation/libparsec/crates/testbed/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "libparsec_testbed"
+version = "0.0.0"
+edition = "2021"
+
+[dependencies]
+libparsec_crypto = { path = "../crypto" }
+libparsec_types = { path = "../types" }
+libparsec_client_types = { path = "../client_types" }
+
+hex-literal = "0.3.3"
+crc32fast = "1.3.2"
+serde = { version = "1.0.147", features = ["derive"] }
+rmp-serde = "1.1.1"
+lazy_static = "1.4.0"
+reqwest = { version = "0.11.13", features = ["native-tls-vendored"] }
```

Comment:
Why the explicit feature is needed ?

Feature unification with crate `libparsec_client_connection` makes reqwest have this feature enabled no matter what

Reply:
It's more explicit

Reply:
We don't care if tls is vendored here (given this is crate that is never shipped in release), we just want reqwest.
On the other hand we don't want this feature to clash with the ones set in `libparsec_platform_client_connection` where the portability question is actually important.

For instance: does `native-tls-vendored` feature available when building for web ? do we want/need it on windows where TLS support is not provided by openssl ?
Those are question we don't need to answer here but in `libparsec_platform_client_connection`

Reply:
Done